botdemo/app/src/main/java/nanorep/com/botdemo/MainActivity.kt

```diff
@@ -204,15 +221,15 @@ class MainActivity : AppCompatActivity(), ChatHandler {
             )
             .datestamp(true, FriendlyDatestampFormatFactory(this))
 
-        if (readoutEnabled) {
-            settings.enableReadOut()
-        }
-
         return ChatController.Builder(this).apply {
             conversationSettings(settings)
             chatEventListener(this@MainActivity)
-            chatHandoverHandler(myHandoverHandler);
-            entitiesProvider(entitiesProvider).apply {  }
+            chatHandoverHandler(handoverHandler);
+            entitiesProvider(entitiesProvider)
+            if (readoutEnabled) {
```

Comment:
where is it coming from ? - if this is a configuration issue - we're checking this on the handlers - don't we?

Reply:
Its an option at the ui (checkbox)